examples/qml_features/rust/src/invokables.rs

```diff
@@ -28,6 +28,7 @@ pub mod ffi {
         }
     }
 
+    // ANCHOR: book_impl_qobject
     impl qobject::RustInvokables {
```

Comment:
```suggestion
    impl qobject::HasInvokables {
```

This type name is a little strange. It implies that the type itself *is* invokables, rather than it *has* invokables.

Reply:
Also, a bit out of scope for this PR, but using floats for RGB values is kinda strange.

Reply:
Ah yes I was originally using `Qt.rgba` in QML which uses floats and then QColor on the Rust side that uses ints (yey for consistency). It could now be changed to be all i32's from 0 -> 255.